azureLogIngestion/adapter/__snapshots__/index.test.ts.snap

```diff
@@ -1,5 +1,111 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`Adapter processes AppPerformanceCounters as metrics 1`] = `Array []`;
+
+exports[`Adapter processes AppPerformanceCounters as metrics 2`] = `
+Array [
+  CountMetric {
+    "attributes": Object {
+      "appRoleInstance": "{appRoleInstance Id}",
+      "appRoleName": "{appRoleName}",
+      "billedSize": 815,
+      "clientIp": "0.0.0.0",
+      "clientType": "PC",
+      "hostInstanceId": "{hostInstanceId}",
+      "instance": "??APP_WIN32_PROC??",
+      "processId": "4608",
+      "resourceId": "/SUBSCRIPTIONS/{subscriptionId}/RESOURCEGROUPS/{functionAppName}/PROVIDERS/MICROSOFT.INSIGHTS/COMPONENTS/{resourceGroupName}",
+      "sdkVersion": "af_azwapccore:2.16.0-18277",
+      "timestamp": 1621887508465,
+    },
+    "interval.ms": undefined,
+    "name": "% Processor Time",
```

Comment:
Isn't this a gauge and not a count?

Reply:
@kolanos It is a gauge and not a count...but apparently in the Node SDK, the gauge type inherits from a generic metric type. If the SDK creates a gauge metric without an intervalMs, it'll be defined on the metric as `undefined`.